src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -34,43 +19,33 @@
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.context.request.NativeWebRequest;
+import org.zalando.problem.spring.web.advice.Responses;
 
 import javax.validation.Valid;
 import java.util.List;
-import java.util.Objects;
 
 import static de.zalando.aruha.nakadi.util.FeatureToggleService.Feature.DISABLE_EVENT_TYPE_CREATION;
 import static de.zalando.aruha.nakadi.util.FeatureToggleService.Feature.DISABLE_EVENT_TYPE_DELETION;
 import static org.springframework.http.ResponseEntity.status;
-import static org.zalando.problem.spring.web.advice.Responses.create;
 
 @RestController
 @RequestMapping(value = "/event-types")
 public class EventTypeController {
 
     private static final Logger LOG = LoggerFactory.getLogger(EventTypeController.class);
 
+    private final EventTypeService eventTypeService;
     private final EventTypeRepository eventTypeRepository;
-    private final TopicRepository topicRepository;
-    private final PartitionResolver partitionResolver;
-    private final Enrichment enrichment;
     private final FeatureToggleService featureToggleService;
-    private final UUIDGenerator uuidGenerator;
 
     @Autowired
-    public EventTypeController(final EventTypeRepository eventTypeRepository,
-                               final TopicRepository topicRepository,
-                               final PartitionResolver partitionResolver,
-                               final Enrichment enrichment,
-                               final FeatureToggleService featureToggleService,
-                               final UUIDGenerator uuidGenerator)
+    public EventTypeController(final EventTypeService eventTypeService,
+                               final EventTypeRepository eventTypeRepository,
```

Comment:
we have it now here only because of `eventTypeRepository.list()`
Can this be also moved to service layer?
Then the controller will have one dependency less.